src/main/java/ubb/proiectColectiv/businessmanagementbackend/service/UserService.java

```diff
@@ -24,17 +24,21 @@ public String login(String email, String password) {
         if (userInDatabase != null) {
             User user = mapper.convertValue(userInDatabase, User.class); //convert HashMap to User POJO
             if (password.equals(user.getPassword())) {
+
                 if (user.getBlockedStatus().equals(true)) {
                     lastLoginAttempts.put(email, System.currentTimeMillis()); //set time of last login attempt for current user
                     return "BLOCKED";
                 }
+
+                if (user.getFailedLoginCounter() != 0)
```

Comment:
Redundant if. You can set it on 0 no matter what. 
Also, you forgot to add { } for if, like it's written in the standards :P

Reply:
Monsieur Crispy, I accepted your critique and decided to take matters in my own hands. The brackets problem has been dealt with.

Reply:
I bow down in front of you, please merge my request, Sir